recipe/meta.yaml

```diff
@@ -1,21 +1,23 @@
-{% set version = "1.19.2" %}
+{% set version = "1.20.1" %}
 
 package:
   name: numpy_and_numpy_base
   version: {{ version }}
 
 source:
   url: https://github.com/numpy/numpy/releases/download/v{{ version }}/numpy-{{ version }}.tar.gz
-  sha256: 74d0cf50aa28af81874aca3e67560945afd783b2a006913577d6cddc35a824a6
+  sha256: 9bf51d69ebb4ca9239e55bedc2185fe2c0ec222da0adee7ece4125414676846d
   patches:
-    - 0001-Obtain-and-prefer-custom-gfortran-from-env-variable.patch
-    - 0002-intel_mkl-version.patch  # [blas_impl == "mkl"]
-    - 0003-intel_init_mkl.patch  # [blas_impl == "mkl"]
-    - 0004-disable-autorun-for-cmd-test.patch   # [win]
+    - patches/0001-Obtain-and-prefer-custom-gfortran-from-env-variable.patch
+    - patches/0002-intel_mkl-version.patch              # [blas_impl == "mkl"]
+    - patches/0003-intel_init_mkl.patch                 # [blas_impl == "mkl"]
+    - patches/0004-disable-autorun-for-cmd-test.patch   # [win]
+    - patches/0005-array_coercion_fix.patch             # [blas_impl == "mkl" and win]
+    - patches/0006-popcnt_fix.patch                     # [blas_impl == "mkl" and win]
 
 build:
   number: 0
-  skip: True  # [(blas_impl == 'openblas' and win) or py2k]
+  skip: True  # [(blas_impl == 'openblas' and win) or py2k or py<37]
```

Comment:
could you please add a comment on why we disable py<37 here?

Reply:
Added a few lines above the `skip`. Please let me know if any more detail should be provided.